### PR TITLE
Templates for VADS icon and tokens follow up work 

### PR DESCRIPTION
**Pull request by narin**
## Description of Change

Adds 2 issue templates for follow-up work to update our package with VADS token or icon updates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/vads-icon.md
Comment (jessicawoodin) [inline .github/ISSUE_TEMPLATE/vads-icon.md:1]: - Can we add a final step to “Notify @jessicawoodin to add new icons in Figma”?
- Do we need to add a step to update Storybook with the new icon?

**Review by jessicawoodin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/vads-icon.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Acceptance Criteria
+<!-- Add a checkbox for each item required to fulfill the user story/issue. -->  
+
+- [ ] Update [component-library](https://www.npmjs.com/package/@department-of-veterans-affairs/component-library) in `assets/package.json` to the latest version
+- [ ] Run the [extract-icons](https://github.com/department-of-veterans-affairs/va-mobile-library/blob/main/packages/assets/extract-svgs.js) script to extract the individual SVGs 
+- [ ] Verify existence of new icon(s) and add it to the repo
+- [ ] Publish a new version of the assets package
+- [ ] Update components package to incorporate updated assets package
+- [ ] Run [buildIconListFromAssets.js script](https://github.com/department-of-veterans-affairs/va-mobile-library/blob/main/packages/components/buildIconListFromAssets.js) to regenerate assets list
+- [ ] Verify new icon(s) is displaying in Storybook
```
Comment (TimRoe) [inline .github/ISSUE_TEMPLATE/vads-icon.md:26]:
```suggestion
- [ ] Verify new icon(s) available in Storybook
```

File: .github/ISSUE_TEMPLATE/vads-icon.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] [Publish](https://github.com/department-of-veterans-affairs/va-mobile-library/actions/workflows/publish.yml) a new version of the components package
+- [ ] Notify @jessicawoodin to add new tokens to Figma
```
Comment (TimRoe) [inline .github/ISSUE_TEMPLATE/vads-icon.md:28]:
```suggestion
- [ ] Notify @jessicawoodin to add new assets to Figma
```

**Review by TimRoe — APPROVED** [review]
Some minor tweak suggestions, but mostly looks good to me.

**Pull request merged**